src/bootstrap.php

```diff
@@ -676,14 +676,14 @@ function (Request $request, string $number, int $version) use ($app) {
             throw new NotAcceptableHttpException('This article VoR requires version 3.');
         }
 
-        if ('36258' === $number && 'poa' === $articleVersion['status'] && $type->getParameter('version') < 2) {
-            throw new NotAcceptableHttpException('This article PoA requires version 2.');
+        if ('vor' === $articleVersion['status'] && $type->getParameter('version') < 3) {
+            $headers['Warning'] = sprintf('299 elifesciences.org "Deprecation: Support for version %d will be removed"', $type->getParameter('version'));
```

Comment:
what is '299' here?

Reply:
> what is '299' here?

https://tools.ietf.org/html/rfc7234#section-5.5.7